config/FindVIGRANUMPY_DEPENDENCIES.cmake

```diff
@@ -92,9 +92,12 @@ IF(PYTHONINTERP_FOUND)
     # boost_python for the wrong Python version
     IF(Boost_FOUND)
         FIND_LIBRARY(Boost_PYTHON_LIBRARY
-                     NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}   # Linux with multiple Python versions
-                           boost_python-mt                                                 # Mac
-                           boost_python                                                    # default
+                     NAMES boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}      # Linux with multiple Python versions
+                           boost_python-py${PYTHON_VERSION_MAJOR}${PYTHON_VERSION_MINOR}-mt   # Linux with multiple Python versions (multithreaded)
+                           boost_python${PYTHON_VERSION_MAJOR}                                # Mac with Python 3
+                           boost_python${PYTHON_VERSION_MAJOR}-mt                             # Mac with Python 3 (multithreaded)
+                           boost_python                                                       # default
+                           boost_python-mt                                                    # Mac with Python (multithreaded)
```

Comment:
The cmake documentation says:
```
Boost_USE_MULTITHREADED  - Set to OFF to use the non-multithreaded
                           libraries ('mt' tag).  Default is ON.
```
This looks like a mistake. I think the non-multithreaded libraries do *not* have the 'mt' tag (the converse is not true: if the library name doesn't contain the 'mt' tag, it can still be multi-threaded). 'Multi-threaded' here actually means 'thread-safe' (see http://www.boost.org/build/tutorial.html), so one would always prefer the 'mt' tag when the program might use multiple threads.